concepticondata/conceptlists.tsv

```diff
@@ -197,3 +197,4 @@ Smith-2015-97	Smith, A.	2015		91	basic,areal	English	Borneo	A 97 item wordlist d
 Laycock-1968-50	Laycock, D.C.	1968		50	basic,areal	English	New Guinea	A 50 item wordlist	Laycock1968	Laycock1968		51-363	
 Voorhoeve-1971-125	Voorhoeve, C.L.	1971		125	basic,areal	English	New Guinea	A 125 item wordlist survey of West Irian	Voorhoeve1971	Voorhoeve1971		98-114	
 Larson-1972-68	Larson, G.F. AND Larson, M.O.	1972		68	basic,areal	English	New Guinea	A 68 item wordlist based on [Swadesh 100](:ref:Swadesh-1955-100).	Larson1972	Larson1972		90-92	
+Heeschen-1978-104	Heeschen, V.	1978		104	basic,areal	English	New Guinea	A 68 item wordlist based on [Bromley 1967](:ref:Bromley-1967-111).	Heeschen1978	Heeschen1978		41-44	
```

Comment:
at some point, we should make some network of all our internal "refs" in this field. It is an interesting difference between what people SAY their list is based upon, and between what people BASE their list on...